doc/schema/schema.md

```diff
@@ -66,18 +66,28 @@ Moreover, each group contains the following attributes:
 Samples from the posterior distribution p(theta|y).
 
 ### `sample_stats`
-Information and diagnostics for each `posterior` sample, provided by the inference backend. It may vary depending on the algorithm used by the backend (i.e. an affine invariant sampler has no energy associated). The name convention used for `sample_stats` variables is the following:
-* `lp`: (unnormalized) log probability for sample
-* `step_size`
-* `step_size_bar`
-* `tune`: boolean variable indicating if the sampler is tuning or sampling
-* `depth`:
-* `tree_size`:
-* `mean_tree_accept`:
-* `diverging`: HMC-NUTS only, boolean variable indicating divergent transitions
-* `energy`: HMC-NUTS only
-* `energy_error`
-* `max_energy_error`
+Information and diagnostics for each `posterior` sample, provided by the inference
+backend. It may vary depending on the algorithm used by the backend (i.e. an affine
+invariant sampler has no energy associated). Therefore none of this parameters
+should be assumed to be present in the `sample_stats` group. The convention
+below serves to ensure that _if_ a variable is present with one of these names
+it will correspond to the definition included here.
+
+The name convention used for `sample_stats` variables is the following:
+
+* `lp`: The log posterior density (up to an additive constant).
+* `acceptance_rate`: The average acceptance probabilities of all possible samples in the proposed tree.
+* `step_size`: The current integration step size.
+* `step_size_nom`: The nominal integration step size. The `step_size` may differ from this, for example if the step size is jittered. Should only be present if `step_size` is also present and it varies between samples (i.e. step size is jittered).
+* `tree_depth`: The number of tree doublings in the balanced binary tree.
+* `n_steps`: The number of leapfrog steps computed.
```

Comment:
I am not not completely sure about the equivalencies in the table above. Not sure if the relation should be `n_steps = 2^tree_depth` or `n_steps <= 2^tree_dept`. 

So far I think that in Stan the relation is `=`, in PyMC3 it's `<=`, no idea in Turing, in NumPyro only `n_steps` exists and `tree_depth` is calculated manually in `io_numpyro` so the relation is `=` and none of the two exist in Pyro.
@

Reply:
Should be straightforward to empirically verify. I'll do so for Turing.

Reply:
So Turing seems to have `n_steps <= 2^(tree_depth + 1)`.

Reply:
Empirically confirmed that PyMC3 has `n_steps < 2^(tree_depth)` and Stan `n_steps < 2^(tree_depth + 1)`. 

Moreover, in both cases `n_steps` is always a power of 2 minus one: 3, 7, 15...